lib/context_adapter_helper.js

```diff
@@ -23,7 +23,9 @@
 
 'use strict';
 
+var caLogger = require('logops');
 var caConfig = require('./context_adapter_configuration');
+caLogger.setLevel(caConfig.LOGOPS_LEVEL);
```

Comment:
Please, group all the requires in a single var statement, and don't mix code with declarations: move this statement to a line after all the var statements. 

On the other hand, setting the level in the root of a module may cause behaviors difficult to predict, as it is not easy to tell when this particular module will be loaded (as it will depend on the modules it depends from). I suggest to do it in an initialization function somewhere in the main file (or in a function you know when it will be executed, e.g.: the creation of the HTTP server).